### DISCUSSION TITLE
Text position animation as a marquee/horizontal news feed/scrolling text

### DISCUSSION THREAD

**Discussion by Nosferatu31** [fabricjs/fabric.js#9410] [Show and tell]
As stated [here](https://github.com/fabricjs/fabric.js/issues/3050), is there any jsfiddle/reproduction or demo to achieve the marquee effect on text?

![https://media3.giphy.com/media/H1SM6E3YiR9DnEd7NJ/giphy.gif](https://media3.giphy.com/media/H1SM6E3YiR9DnEd7NJ/giphy.gif)

**Comment by ShaMan123** [comment]
I would animte the position of the text and add a clipPath that is absolutePositioned or put the text in a group and clip the group while setting the group layout to fixed.
I am sure there are other ways

**Reply by ShaMan123** [reply]
If the text animates across the entire canvas viewport clipping is redundant

**Reply by Nosferatu31** [reply]
I think I managed to achieve it:
https://codesandbox.io/s/fabric-marquee-text-nmps73

**Reply by ShaMan123** [reply]
very nice, this means that scrolling can be achieved
